linux_os/guide/system/selinux/package_libselinux_installed/rule.yml

```diff
@@ -0,0 +1,38 @@
+documentation_complete: true
+
+prodtype: fedora,ol7,ol8,rhel7,rhel8,rhv4,ocp4
+
+title: 'Install libselinux Package'
+
+description: |-
+    {{{ describe_package_install(package="libselinux") }}}
+
+rationale: |-
+    Security-enhanced Linux is a feature of the Linux kernel and a number of utilities
+    with enhanced security functionality designed to add mandatory access controls to Linux.
+    The Security-enhanced Linux kernel contains new architectural components originally
+    developed to improve security of the Flask operating system. These architectural components
```

Comment:
Please remove this entire sentence mentioning Flash. It is not needed in the rationale at all and will serve to create unnecessary confusion.

Reply:
OK, makes sense. Done.